1/contrib/jenkins/kube-slave-common.sh

```diff
@@ -83,17 +85,30 @@ function generate_kubernetes_config() {
     else
       return
     fi
-    [ -z "${slave_templates}" ] && return
     echo "
-    <org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud plugin=\"kubernetes@0.5\">
+    <org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud plugin=\"kubernetes@0.6\">
```

Comment:
we should minimally add a comment in base-plugins.txt to update this line when we bump the kubernetes plugin version there.

extra credit would be to somehow grep/sed the version from base-plugin.txt and inject it into this echo